safeeyes/safeeyes/config/lang/de.json

```diff
@@ -0,0 +1,45 @@
+{
+    "meta_info": {
+        "language_name": "Deutsch",
+        "language_name_en": "German"
+    },
+    "exercises": {
+        "long_break_exercises": [
+            "Laufen Sie ein wenig",
+            "Lehnen Sie sich zurück und entspannen Sie sich"
+        ],
+        "short_break_exercises": [
+            "Schließen Sie die Augen",
+            "Rollen Sie mit den Augen",
+            "Rotieren Sie mit den Augen im Uhrzeigersinn",
+            "Rotieren Sie mit den Augen gegen den Uhrzeigersinn",
+            "Zwinkern Sie mit den Augen",
+            "Fokussieren Sie sich auf einen weit entfernten Punkt",
+            "Trinken Sie ein wenig Wasser"
+        ]
+    },
+    "messages": {
+        "ready_for_a_break": "Sind Sie bereit für eine Pause in {} Sekunden",
```

Comment:
Google translates it as "Are you ready for a break in {} seconds?". If this is actually in a question format could you please change it to a command/statement like "Be ready for a break in {} seconds"